wvdecrypter/wvdecrypter_android_jni.cpp

```diff
@@ -564,48 +570,61 @@ bool WV_CencSingleSampleDecrypter::KeyUpdateRequest(bool waitKeys)
     {
       Log(SSD_HOST::LL_INFO, "Key request not successful - trying provisioning");
       provisionRequested = true;
-      return KeyUpdateRequest(waitKeys);
+      return GetKeyRequest(keyRequestData);
     }
     else
       Log(SSD_HOST::LL_ERROR, "Key request not successful");
     return false;
   }
 
-  pssh_.clear();
-  optParams_.clear();
-
-  std::vector<char> keyRequestData = keyRequest.getData();
+  keyRequestData = keyRequest.getData();
   Log(SSD_HOST::LL_DEBUG, "Key request successful size: %lu", keyRequestData.size());
+  return true;
+}
 
-  if (!SendSessionMessage(keyRequestData))
+bool WV_CencSingleSampleDecrypter::KeyUpdateRequest(bool waitKeys, bool sendSessionMsg /* = true */)
```

Comment:
comment left over from debugging?

Reply:
i have add the `/* = true */` comment the same way code is handled in Kodi sources
to make more easy to understand when an method argument has a default value set